### PR TITLE
tidb-lightning: clarify an ambiguous description for `sorted-kv-dir`

### PR DESCRIPTION
**Pull request by TomShawn**
### What is changed, added or deleted? (Required)
Clarify an ambiguous description for `sorted-kv-dir`.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [x] v6.0 (TiDB 6.0 versions)
- [x] v5.4 (TiDB 5.4 versions)
- [x] v5.3 (TiDB 5.3 versions)
- [x] v5.2 (TiDB 5.2 versions)
- [x] v5.1 (TiDB 5.1 versions)
- [x] v5.0 (TiDB 5.0 versions)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by sunzhaoyang — APPROVED** [review]
LGTM

File: tidb-lightning/deploy-tidb-lightning.md [hunk 1]
```diff
@@ -81,7 +81,9 @@ Refer to the [TiDB enterprise tools download page](/download-ecosystem-tools.md#
 
 2. Mount the data source onto the same machine.
 
-3. Configure `tidb-lightning.toml`. For configurations that do not appear in the template below, TiDB Lightning writes a configuration error to the log file and exits. `sorted-kv-dir` must be an empty directory and the disk where the directory is located must have a lot of free space.
+3. Configure `tidb-lightning.toml`. For configurations that do not appear in the template below, TiDB Lightning writes a configuration error to the log file and exits.
+
+    `sorted-kv-dir` sets the temporary storage directory for the sorted Key-Value files. The directory must be empty, and the storage space must be enough to hold the largest single table in the data source. See [Downstream storage space requirements](/tidb-lightning/tidb-lightning-requirements.md#resource-requirements) for details.
```
Comment (lonng) [inline tidb-lightning/deploy-tidb-lightning.md:86]: Can we highlight the sentence： `must be enough to hold the largest single table in the data source`.

**Review by shichun-0415 — APPROVED** [review]
LGTM

File: tidb-lightning/deploy-tidb-lightning.md (hunk 1 above)
Comment (TomShawn) [reply tidb-lightning/deploy-tidb-lightning.md:86]:
```suggestion
    `sorted-kv-dir` sets the temporary storage directory for the sorted Key-Value files. The directory must be empty, and the storage space **must be enough to hold the largest single table in the data source**. See [Downstream storage space requirements](/tidb-lightning/tidb-lightning-requirements.md#resource-requirements) for details.
```

File: migrate-aurora-to-tidb.md [hunk 2]
```diff
@@ -105,7 +105,7 @@ pd-addr = "${ip}:${port}"     # The cluster PD address, e.g.: 172.16.31.3:2379.
 # "tidb": The "tidb" backend is recommended to import data less than 1 TiB. During the import, the target TiDB cluster can provide service normally.
 backend = "local"
 
-# Set the temporary storage directory for the sorted Key-Value files. The directory must be empty, and the storage space must be enough to hold the largest single table in the data source. For better import performance, it is recommended to use a directory different from `data-source-dir` and use flash storage, which can use I/O exclusively.
+# Set the temporary storage directory for the sorted Key-Value files. The directory must be empty, and the storage space must be large enough to hold the dataset to be imported. For better import performance, it is recommended to use a directory different from `data-source-dir` and use flash storage, which can use I/O exclusively.
```
Comment (TomShawn) [inline migrate-aurora-to-tidb.md:108]:
```suggestion
# Set the temporary storage directory for the sorted Key-Value files. The directory must be empty, and the storage space must be greater than the size of the dataset to be imported. For better import performance, it is recommended to use a directory different from `data-source-dir` and use flash storage, which can use I/O exclusively.
```

File: migrate-from-csv-files-to-tidb.md [hunk 3]
```diff
@@ -64,7 +64,7 @@ file = "tidb-lightning.log"
 # "local": Default backend. The local backend is recommended to import large volumes of data (1 TiB or more). During the import, the target TiDB cluster cannot provide any service.
 # "tidb": The "tidb" backend is recommended to import data less than 1 TiB. During the import, the target TiDB cluster can provide service normally.
 backend = "local"
-# Set the temporary storage directory for the sorted Key-Value files. The directory must be empty, and the storage space must be enough to hold the largest single table from the data source. For better import performance, it is recommended to use a directory different from `data-source-dir` and use flash storage, which can use I/O exclusively.
+# Set the temporary storage directory for the sorted Key-Value files. The directory must be empty, and the storage space must be large enough to hold the dataset to be imported. For better import performance, it is recommended to use a directory different from `data-source-dir` and use flash storage, which can use I/O exclusively.
```
Comment (TomShawn) [inline migrate-from-csv-files-to-tidb.md:67]:
```suggestion
# Set the temporary storage directory for the sorted Key-Value files. The directory must be empty, and the storage space must be greater than the size of the dataset to be imported. For better import performance, it is recommended to use a directory different from `data-source-dir` and use flash storage, which can use I/O exclusively.
```

File: migrate-from-sql-files-to-tidb.md [hunk 4]
```diff
@@ -63,7 +63,7 @@ file = "tidb-lightning.log"
 # "tidb"：The "tidb" backend can also be used to import small volumes of data (less than 1 TiB). During the import, the target TiDB cluster can provide service normally. For the information about backend mode, refer to https://docs.pingcap.com/tidb/stable/tidb-lightning-backends.
 
 backend = "local"
-# Sets the temporary storage directory for the sorted key-value files. The directory must be empty, and the storage space must be enough to store the largest single table from the data source. For better import performance, it is recommended to use a directory different from `data-source-dir` and use flash storage and exclusive I/O for the directory.
+# Sets the temporary storage directory for the sorted key-value files. The directory must be empty, and the storage space must be large enough to hold the dataset to be imported. For better import performance, it is recommended to use a directory different from `data-source-dir` and use flash storage and exclusive I/O for the directory.
```
Comment (TomShawn) [inline migrate-from-sql-files-to-tidb.md:66]:
```suggestion
# Sets the temporary storage directory for the sorted key-value files. The directory must be empty, and the storage space must be greater than the size of the dataset to be imported. For better import performance, it is recommended to use a directory different from `data-source-dir` and use flash storage and exclusive I/O for the directory.
```

File: migrate-large-mysql-shards-to-tidb.md [hunk 5]
```diff
@@ -160,7 +160,7 @@ Follow these steps to start `tidb-lightning`:
     # For more information, see [TiDB Lightning Backends](https://docs.pingcap.com/tidb/stable/tidb-lightning-backends)
     backend = "local"
     # Set the temporary directory for the sorted key value pairs. It must be empty.
-    # The free space must be greater than the largest single table of the data source.
+    # The free space must be large enough to hold the dataset to be imported.
```
Comment (TomShawn) [inline migrate-large-mysql-shards-to-tidb.md:163]:
```suggestion
    # The free space must be greater than the size of the dataset to be imported.
```

File: migrate-large-mysql-to-tidb.md [hunk 6]
```diff
@@ -105,7 +105,7 @@ The target TiKV cluster must have enough disk space to store the imported data.
     # "local": Default backend. The local backend is recommended to import large volumes of data (1 TiB or more). During the import, the target TiDB cluster cannot provide any service.
     # "tidb": The "tidb" backend is recommended to import data less than 1 TiB. During the import, the target TiDB cluster can provide service normally. For more information on the backends, refer to https://docs.pingcap.com/tidb/stable/tidb-lightning-backends.
     backend = "local"
-    # Sets the temporary storage directory for the sorted Key-Value files. The directory must be empty, and the storage space must be enough to hold the largest single table in the data source. For better import performance, it is recommended to use a directory different from `data-source-dir` and use flash storage, which can use I/O exclusively.
+    # Sets the temporary storage directory for the sorted Key-Value files. The directory must be empty, and the storage space must be large enough to hold the dataset to be imported. For better import performance, it is recommended to use a directory different from `data-source-dir` and use flash storage, which can use I/O exclusively.
```
Comment (TomShawn) [inline migrate-large-mysql-to-tidb.md:108]:
```suggestion
    # Sets the temporary storage directory for the sorted Key-Value files. The directory must be empty, and the storage space must be greater than the size of the dataset to be imported. For better import performance, it is recommended to use a directory different from `data-source-dir` and use flash storage, which can use I/O exclusively.
```

File: tidb-lightning/deploy-tidb-lightning.md [hunk 7]
```diff
@@ -81,7 +81,9 @@ Refer to the [TiDB enterprise tools download page](/download-ecosystem-tools.md#
 
 2. Mount the data source onto the same machine.
 
-3. Configure `tidb-lightning.toml`. For configurations that do not appear in the template below, TiDB Lightning writes a configuration error to the log file and exits. `sorted-kv-dir` must be an empty directory and the disk where the directory is located must have a lot of free space.
+3. Configure `tidb-lightning.toml`. For configurations that do not appear in the template below, TiDB Lightning writes a configuration error to the log file and exits.
+
+    `sorted-kv-dir` sets the temporary storage directory for the sorted Key-Value files. The directory must be empty, and the storage space **must be large enough to hold the dataset to be imported**. See [Downstream storage space requirements](/tidb-lightning/tidb-lightning-requirements.md#resource-requirements) for details.
```
Comment (TomShawn) [inline tidb-lightning/deploy-tidb-lightning.md:86]:
```suggestion
    `sorted-kv-dir` sets the temporary storage directory for the sorted Key-Value files. The directory must be empty, and the storage space **must be greater than the size of the dataset to be imported**. See [Downstream storage space requirements](/tidb-lightning/tidb-lightning-requirements.md#resource-requirements) for details.
```

File: tidb-lightning/tidb-lightning-requirements.md [hunk 8]
```diff
@@ -117,4 +117,4 @@ The CPU and memory consumed by TiDB Lightning vary with the backend mode. Run Ti
...
 - Importer-backend: In this mode, resource consumption is nearly the same as that in Local-backend. Importer-backend is not recommended and you are advised to use Local-backend if you have no particular requirements.
 
-**Storage space**: The `sorted-kv-dir` configuration item specifies the temporary storage directory for the sorted key-value files. The directory must be empty, and the storage space must be enough to store the largest single table from the data source. For better import performance, it is recommended to use a directory different from `data-source-dir` and use flash storage and exclusive I/O for the directory.
+**Storage space**: The `sorted-kv-dir` configuration item specifies the temporary storage directory for the sorted key-value files. The directory must be empty, and the storage space must be large enough to hold the dataset to be imported. For better import performance, it is recommended to use a directory different from `data-source-dir` and use flash storage and exclusive I/O for the directory.
```
Comment (TomShawn) [inline tidb-lightning/tidb-lightning-requirements.md:120]:
```suggestion
**Storage space**: The `sorted-kv-dir` configuration item specifies the temporary storage directory for the sorted key-value files. The directory must be empty, and the storage space must be greater than the size of the dataset to be imported. For better import performance, it is recommended to use a directory different from `data-source-dir` and use flash storage and exclusive I/O for the directory.
```

**Review by lonng — APPROVED** [review]

**Pull request merged**